### PR TITLE
feat: add FormatPrice component

### PR DESCRIPTION
**Pull request by Nitish-hack**
### Related Issue 

Closes: #766 

#### Changes made 👷🏻‍♂️
I have created a folder named Helper in src , and in that a file named FormatPrice.jsx, and i have also done changes in Product card

#### Screenshots  📸

image

### CODE REVIEW

File: src/components/Helpers/FormatPrice.jsx
Comment (tamalCodes) [inline src/components/Helpers/FormatPrice.jsx:1]: Shouldn't the extension of the file be just `.js` instead of `.jsx` ?

**Review by tamalCodes — APPROVED** [review]
The rest looks good !

File: src/components/Helpers/FormatPrice.jsx
Comment (Nitish-hack) [reply src/components/Helpers/FormatPrice.jsx:1]: okay, I'll just change the extension and

**Review by tamalCodes — APPROVED** [review]
**Amazing work,**
This looks good to me, will be merged soon 🎉.

**Pull request merged**